assets/eip-7432/ERC7432.sol

```diff
@@ -135,20 +151,20 @@ contract ERC7432 is IERC7432 {
         emit RoleApproval(msg.sender, _tokenAddress, _tokenId, _operator, _approved);
     }
 
-    function getApprovedRole(
-        address _grantor,
+    function isRoleApprovedForAll(
         address _tokenAddress,
-        uint256 _tokenId,
+        address _grantor,
         address _operator
-    ) external view override returns (bool) {
-        return tokenApprovals[_grantor][_tokenAddress][_tokenId][_operator];
+    ) public view override returns (bool) {
+        return operatorApprovals[_grantor][_tokenAddress][_operator];
     }
 
-    function isRoleApprovedForAll(
+    function getApprovedRole(
+        address _tokenAddress,
+        uint256 _tokenId,
         address _grantor,
-        address _operator,
-        address _tokenAddress
-    ) external view override returns (bool) {
-        return operatorApprovals[_grantor][_tokenAddress][_operator];
+        address _operator
+    ) public view override returns (bool) {
+        return tokenApprovals[_grantor][_tokenAddress][_tokenId][_operator];
```

Comment:
mapping `tokenApprovals` was not declared. I would rename it to `tokenIdApprovals`
```suggestion
        return tokenApprovals[_grantor][_tokenAddress][_tokenId][_operator];
```